### PR TITLE
Add api.changeAdminStatus

### PR DESCRIPTION
**Pull request by ravkr**
few days ago I've made similar function to #658
But I didn't have time to write docs so I didn't pushed it to Github
It looks a bit cleaner IMO

I've also done some tests in the wild
and you can change `adminStatus` of every user in thread without being blocked by Facebook

but if you try to do the same thing one by one... Facebook blocks you for an hour :(

obraz

### CODE REVIEW

**Comment by ravkr** [comment]
changelog record and docs added

**Comment by ravkr** [comment]
@bsansouci, could we merge?

File: DOCS.md [hunk 1]
```diff
@@ -22,7 +23,6 @@
 * [`api.getThreadPictures`](#getThreadPictures)
 * [`api.getUserID`](#getUserID)
 * [`api.getUserInfo`](#getUserInfo)
-* [`api.threadColors`](#threadColors)
```
Comment (bsansouci) [inline DOCS.md]: Can you undo this change? (You moved the function down)

File: index.js [hunk 2]
```diff
@@ -75,17 +75,18 @@ function buildAPI(globalOptions, html, jar) {
     },
   };
 
-  var apiFuncNames = [
+  const apiFuncNames = [
     'addUserToGroup',
     'changeArchivedStatus',
     'changeBlockedStatus',
     'changeGroupImage',
+    'changeNickname',
```
Comment (bsansouci) [inline index.js]: Can you undo these moves?

**Comment by ravkr** [comment]
These little changes were just sorting alphabetically
Should I really undo them?

**Comment by Schmavery** [comment]
Alright had a chance to look at this again. If this is the one that you guys want to merge, let's change the name to `changeAdminStatus` to be more consistent with the other function names and then I'm happy to merge. Idrc about the function order, sorry for the delay here but in the future it's probably simplest to keep it to one PR per feature when possible.

**Comment by Schmavery** [comment]
thanks!
